core/validatorapi/router.go

```diff
@@ -126,6 +127,11 @@ func NewRouter(h Handler, eth2Cl eth2client.Service) (*mux.Router, error) {
 			Path:    "/eth/v1/beacon/blinded_blocks",
 			Handler: submitBlindedBlock(h),
 		},
+		{
+			Name:    "submit_validator_registration",
+			Path:    "/eth/v1/validator/register_validator",
+			Handler: SubmitValidatorRegistrations(h),
```

Comment:
The implementation in go-eth2-client is submit registrations as an array for all validators on a given node

https://github.com/attestantio/go-eth2-client/blob/c05328b73d87b8bcca8cab74f97c471fd85e5f3a/http/submitvalidatorregistrations.go